platform/api/gcloud/compute.go

```diff
@@ -52,6 +52,10 @@ func (a *API) mkinstance(userdata, name string, keys []*agent.Key) *compute.Inst
 			Value: &sshKeys,
 		})
 	}
+	nicType := ""
```

Comment:
```suggestion
	nicType := "VIRTIO_NET"
```

What do you think ? 

Doc: https://pkg.go.dev/google.golang.org/api/compute/v1#Instance
```
        // NicType: The type of vNIC to be used on this interface. This may be
	// gVNIC or VirtioNet.
	//
	// Possible values:
	//   "GVNIC" - GVNIC
	//   "UNSPECIFIED_NIC_TYPE" - No type specified.
	//   "VIRTIO_NET" - VIRTIO
	NicType string `json:"nicType,omitempty"`
```

Reply:
Yeah makes sense